### PR TITLE
[SDKS-836] Expose Collector and JailbreakDetector initialization meth…

### PR DESCRIPTION
**Pull request by jhoongo**
…ods as public

### CODE REVIEW

File: FRAuth/FRAuth/Device/HardwareCollector.swift [hunk 1]
```diff
@@ -18,6 +18,9 @@ public class HardwareCollector: DeviceCollector {
     /// Name of current collector
     public var name: String = "hardware"
     
+    /// Initializes BrowserCollector instance
```
Comment (cerebrl) [inline FRAuth/FRAuth/Device/HardwareCollector.swift:21]: I think you meant `HardwareCollector`, rather than `BrowserCollector` :)

**Review by cerebrl — COMMENTED** [review]
Other than the one likely copy-paste issue, this looks good.

File: FRAuth/FRAuth/Device/ProfileCollector.swift [hunk 2]
```diff
@@ -19,7 +19,7 @@ public class ProfileCollector: DeviceCollector {
     public var collectors: [DeviceCollector]
     
     /// Private initialization method which initializes default array of ProfileCollector
```
Comment (spetrov) [inline FRAuth/FRAuth/Device/ProfileCollector.swift:21]: Minor: "Private" should become "public" in comment

**Review by spetrov — APPROVED** [review]
Looks good to me! I only have a minor comment regarding a comment in the code that we may want to update...

File: FRAuth/FRAuth/Device/ProfileCollector.swift (hunk 2 above)
Comment (jhoongo) [reply FRAuth/FRAuth/Device/ProfileCollector.swift]: Thanks Stoyan! Updated the comment accordingly.

File: FRAuth/FRAuth/Device/HardwareCollector.swift (hunk 1 above)
Comment (jhoongo) [reply FRAuth/FRAuth/Device/HardwareCollector.swift]: Thanks Justin, I've updated the inline comment. :)

**Review by spetrov — APPROVED** [review]
👍

**Review by cerebrl — APPROVED** [review]

**Review by george-bafaloukas-forgerock — APPROVED** [review]
Looks good

**Pull request merged**